src/web/app/pages-instructor/instructor-session-individual-extension-page/instructor-session-individual-extension-page.component.html

```diff
@@ -0,0 +1,203 @@
+<tm-loading-retry [shouldShowRetry]="hasLoadingFeedbackSessionFailed" [message]="'Failed to load course and feedback session'" (retryEvent)="loadFeedbackSessionAndIndividuals()">
+  <div *tmIsLoading="isLoadingFeedbackSession">
+    <div class="card border-primary margin-top-20px">
+      <div class="card-body">
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Course ID</div>
+          <div id="time-zone" class="col-md-3 text-md-left">
+            {{ courseId }}
+          </div>
+        </div>
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Time Zone</div>
+          <div id="time-zone" class="col-md-3 text-md-left">
+            {{ feedbackSessionTimeZone }}
+          </div>
+        </div>
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Course Name</div>
+          <div id="course-name" class="col-md-10 text-md-left">
+            {{ courseName }}
+          </div>
+        </div>
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Session Name</div>
+          <div id="course-name" class="col-md-10 text-md-left">
+            {{ feedbackSessionName }}
+          </div>
+        </div>
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Original Deadline</div>
+          <div id="course-name" class="col-md-10 text-md-left">
+            {{ timezoneService.formatToString(feedbackSessionEndingTime, feedbackSessionTimeZone, DATETIME_FORMAT) }}
```

Comment:
It could be better to perform this formatting in the logic instead, and only after we've fetched the feedback session information - if the arguments are still their default values (before we've fetched the info), errors can show up

<img width="581" alt="Screenshot 2022-03-20 at 3 16 35 PM" src="https://user-images.githubusercontent.com/41856541/159152372-e5080164-81f0-4cd8-a5bf-4998046c3a55.png">

Reply:
The problem was that timezone default's was ''. I've put the default as 'UTC'. Because loading only stops after the correct zone is fetched, should not be a problem